_content/references.md

```diff
@@ -1,5 +1,170 @@
+| Agora Interactive Whiteboard                 | Interactive Whiteboard              | Provides a simple collaboration platform on a virtual whiteboard where users from different locations can draw, annotate, and share PDF documents. |
+| Agora, Inc.                                  | Agora                               | Official company name.                                       |
+| Agora.io                                     | Agora                               | Variant of company name. Not used in technical documentation, with the exception of copyright statements (for example, "© 2016 Agora.io. All rights reserved.") |
+
+#### A
+
+- **add-on** (noun, adjective), **add on** (verb)
+- **Android device**
+- **app** (lowercase except when beginning a sentence or used as a proper noun)
+- **audio sample**: Use instead of *audio capture*.
+- **audio sample rate**
+
+#### B
+
+- **backend**: Use instead of *back-end* or *back end*.
+- **beta**
+- **bitrate**
+- **Boolean** (uppercase when referring to the datatype)
+- **Bluetooth**
+- **bridging header file**
+
+#### C
+
+- **case-sensitive**
+
+#### D
+
+- **data packet**
+- **dual-stream mode**
+- **dual stream** (when not modifying a noun)
+
+#### E
+
+- **email**
+- **earlier**: Use instead of *lower* when describing a version.
+- **extract**: Use instead of *unarchive* or *uncompress*.
+
+#### F
+
+- **filename**
+- **fill in; fill out**: Use *fill in* when referring to entering information in individual fields. Use *fill out* when referring to completing an entire form.
+- **front camera**
+
+#### G
+
+- **get**: Use instead of *fetch* or *retrieve*.
+
+#### H
+
+- **HTML5**: Use instead of *H5*.
+
+#### I
+
+- **incorrect**: Use instead of *wrong*.
+- **image enhancement**
+- **internet**
+
+#### J
+
+- **join a channel**: Use instead of *enter a channel*.
+- **just**: Avoid. Usually, *just* is a filler word that you can delete without affecting your meaning.
+
+#### L
+
+- **later**: Use instead of *higher* when describing a version.
+- **live stream** (verb or noun)
+- **log in** (verb), **login** (noun or adjective)
+
+#### M
+
+- **might**: Use instead of may when referring to possibility.
+- **multichannel**
+- **multimedia**
+- **multithreaded**
+
+#### N
+
+- **number**, instead of *no.*
+
+#### O
+
+- **open source** (no hyphen, not even as an adjective or verb)
+
+#### P
+
+- **packet loss rate**
+- **path**: Avoid using *filepath*, *file path*, *pathname*, or *path name* if possible.
+- **plugin** (noun), **plug-in** (adjective), **plug in** (verb)
+
+#### R
+
+- **real time** (noun or adverb), **real-time** (adjective)
+- **reintegrate**
+
+#### S
+
+- **screen sharing**
+- **set up** (verb), **setup** (noun)
+- **show**: Use instead of *demonstrate*.
+- **signup** (verb), **sign-up** (noun)
```

Comment:
As in signup Agora, or signup for an Agora account?

Reply:
Should be sign up for an Agora account.